app/views/membership.html

```diff
@@ -281,7 +281,11 @@
                 <div class="col-add-role" row>
                   <div class="checkbox">
                     <label>
-                      <input type="checkbox" class="toggle-hidden" ng-click="showAllRoles($event)">
+                      <input
+                        type="checkbox"
+                        class="toggle-hidden"
+                        ng-click="showAllRoles($event)"
```

Comment:
Rename `showAllRoles()` to `toggleRoles()`? `showAllRoles` to me means always show them.